src/components/ButtonPanel.css

```diff
@@ -1,3 +1,12 @@
-.ButtonPanel {
+.button-panel {
   text-align: center;
```

Comment:
**OPTIONAL**: Is this necessary? I see you are also centering on the `.button` class.

Reply:
Removed